index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property="og:image" content="./images/bird.jpg">
+    <meta property="og:title" content="Linkbrary">
+    <meta 
+      property="og:description"
+      content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요"
+    >
+    <meta property="og:url" content="https://www.codeit.kr">
+    <title>Linkbrary</title>
+    <link rel="stylesheet" href="./css/colors.css">
+    <link rel="stylesheet" type="text/css" href="./css/reset.css">
+    <link rel="stylesheet" href="./css/landing.css">
+  </head>
+  <body>
+    <nav>
+      <div class="navbar">
+        <a href="codeit.html">
+          <img class="logo-image" src="./images/logo.svg" alt="Linkbrary 로고">
+        </a>
+        <a class="login-button" href="signin.html">
+          <span>로그인</span>
+        </a>
+      </div>
+    </nav>
+    <header>
+      <div class="header-section">
+        <h1 class="slogan">
+          <span class="slogan-text">세상의 모든 정보</span><span>를<br>쉽게 저장하고 관리해 보세요</span>
+        </h1>  
+        <a class="CTA" href="signup.html">
+          <span>링크 추가하기</span>
+        </a>
+        <img class="header-section-img" src="./images/header-section-img.png" alt="Linkbrary 예시 이미지">
+      </div>
+    </header>
+    <main>
+      <div class="section">
+        <div class="section-container">
+          <h2 class="section-title">   
+            <span class="section-color-title-1">원하는 링크</span><span>를<br>저장하세요</span>
+          </h2>
+          <p class="description">
+            나중에 읽고 싶은 글, 다시 보고 싶은 영상,<br> 사고 싶은 옷, 기억하고 싶은 모든 것을<br> 한 공간에 저장하세요.
+          </p>
+        </div>
+        <img class="section-img" src="./images/section-title-img1.png" alt="원하는 링크 저장 예시 이미지">
+      </div>
+      <div class="section reverse">
+        <img class="section-img" src="./images/section-title-img2.png" alt="링크를 폴더로 관리 예시 이미지">
+        <div class="section-container">
+          <h2 class="section-title">
+            <span>링크를 폴더로</span><br>
+            <span class="section-color-title-2">관리</span><span>하세요</span>
+          </h2>
+          <p class="description">
+            나만의 폴더를무제한으로 만들고<br> 다양하게 활용할 수 있습니다.
```

Comment:
띄어쓰기, 공백 확인해주세요~
```suggestion
            나만의 폴더를 무제한으로 만들고<br>다양하게 활용할 수 있습니다.
```

여기 외에도 텍스트 들어간 부분들 전체적으로 확인해주세요!